sdk/servicebus/Azure.Messaging.ServiceBus/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 # Release History
 
-## 7.0.0-preview.9 (Unreleased)
+## 7.0.0-preview.9 (2020-11-04)
 
+### Added
+- Added dependency on Azure.Core.Amqp library.
+- Added dependency on System.Memory.Data library.
+
+### Breaking Changes
+- Removed `AmqpMessage` property in favor of a `GetRawMessage` method on `ServiceBusMessage` and `ServiceBusReceivedMessage`.
```

Comment:
I assume these breaks are from the last preview version and not the last GA version correct?

Reply:
This is still a preview.